packages/element-templates-json-schema-shared/src/defs/base.json

```diff
@@ -50,6 +50,27 @@
         ]
       }
     },
+    "elementType": {
+      "$id": "#/elementType",
+      "type": "object",
+      "description": "The BPMN type the element will be transformed into.",
+      "default": {},
+      "properties": {
+        "value": {
+          "$id": "#/elementType/value",
+          "type": "string",
+          "pattern": "^(.*?:)",
```

Comment:
* Why the group `()`? Do we re-use it?
* Why any character? `.` should it not be rather `\w` or maybe `[\w\d]`? So to not have whitespaces allowed?
* Why `*` (0..any) and not `+` (1...any)
* Why not also require to have something behind the `:` ?

For example what about
`^[\w\d]+:[\w\d]+$`

Reply:
All valid improvements that I did not consider - I just copied the validation from `appliesTo`: https://github.com/camunda/element-templates-json-schema/blob/fbb61b3396b90633e8d13966d16404547627b589/packages/element-templates-json-schema-shared/src/defs/base.json#L40-L42

I will improve both RegEx with your suggestions